src/tests/tests_runtime.rs

```diff
@@ -351,7 +351,34 @@ fn test_get_current_period_and_thread() {
              println!("Module type WasmV1Module");
          }
      }
-     run_main(&*interface, runtime_module, 100_000, gas_costs).unwrap();
+     run_main(&*interface, runtime_module, 100_000_000, gas_costs).unwrap();
+}
+
+#[test]
+#[serial]
+/// Test test_native_hash
+fn test_native_hash() {
```

Comment:
you may want to add _wasmv1_as to the name of the test so we know that it's the new abi and the sc is originally written in as (later will have rs)